streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -425,6 +427,9 @@ public void flushInternal() {
     @Override
     public void close() {
         flush();
+        options.dispose();
+        wOptions.dispose();
+        fOptions.dispose();
```

Comment:
I think `db` is sufficient to trigger NPE upon unexpected usage after `close`, I have other objects defined `final` and would prefer not to change it; another way is to have an `isClosed` flag and check that flag on all operations.